src/simplewallet/simplewallet.cpp

```diff
@@ -3108,12 +3099,7 @@ bool simple_wallet::sweep_main(uint64_t below, const std::vector<std::string> &a
     }
     else while (!ptx_vector.empty())
```

Comment:
Is this test supposed to be around now ?

Reply:
Good catch, thanks!